__tests__/component/form/CreateTimesheetForm.test.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react"
+import { render, screen, fireEvent, waitFor } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import sinon, { spy } from "sinon"
+import { Customer, Employee, Project, Timesheet } from "@/lib/types/apiTypes"
+import { ApiUpdateResponse } from "@/lib/types/hooks"
+import { CreateTimesheetForm } from "@/components/form/TimesheetForm"
+import { NEXT_PUBLIC_API_URL } from "@/lib/conf"
+
+// eslint-disable-next-line id-match, id-length
+import _ from "lodash"
+
+const customerEndpointURL = `${NEXT_PUBLIC_API_URL}/timesheet`
+const bodySpy = sinon.spy((body) => body)
+const pathSpy = sinon.spy((path) => path)
+
+jest.mock("@/lib/utils/fetch", () => ({
+    // eslint-disable-next-line require-await
+    post: async (path: string, body: object): Promise<ApiUpdateResponse<Timesheet>> => pathSpy(path) && bodySpy(body),
+}))
+
+// eslint-disable-next-line camelcase
+const testEmployee: Employee = { id: 1, first_name: "testi", last_name: "tekija", email: "test@test.test" }
+
+const testProject: Project = {
```

Comment:
You could create a file called `fixtures.ts` that exports these. The following fixtures from the `CreateCustomerForm` test could be moved to the same file:
```typescript
const testCustomerRequiredFields: Customer = { name: "Name" }
const testCustomerAllFields: Customer = { name: "anotherName", description: "description" }
```